### ISSUE TITLE
[Beta] Applied Accessibility - Test fails to check for closing </footer> tag

### ISSUE THREAD

**Issue by Asjas** [freeCodeCamp/freeCodeCamp#16416]
#### Challenge Name

https://beta.freecodecamp.org/en/challenges/applied-accessibility/make-screen-reader-navigation-easier-with-the-footer-landmark

#### Issue Description

The test passes if the opening `<footer>` tag is used and the closing `</footer>` is missing.

Also, the test states the no `<div>` tag may be used in the code but it doesn't check if the closing `</div>` tag is still left in.

#### Browser Information

* Browser Name, Version: Chrome, Version 63.0.3239.108 (Official Build) (64-bit)
* Operating System: Windows 10 Pro
* Mobile, Desktop, or Tablet: Desktop

#### Your Code

```js
<body>
  <header>
    <h1>Training</h1>
    <nav>
      <ul>
        <li><a href="#stealth">Stealth &amp; Agility</a></li>
        <li><a href="#combat">Combat</a></li>
        <li><a href="#weapons">Weapons</a></li>
      </ul>
    </nav>
  </header>
  <main>
    <section id="stealth">
      <h2>Stealth &amp; Agility Training</h2>
      <article><h3>Climb foliage quickly using a minimum spanning tree approach</h3></article>
      <article><h3>No training is NP-complete without parkour</h3></article>
    </section>
    <section id="combat">
      <h2>Combat Training</h2>
      <article><h3>Dispatch multiple enemies with multithreaded tactics</h3></article>
      <article><h3>Goodbye world: 5 proven ways to knock out an opponent</h3></article>
    </section>
    <section id="weapons">
      <h2>Weapons Training</h2>
      <article><h3>Swords: the best tool to literally divide and conquer</h3></article>
      <article><h3>Breadth-first or depth-first in multi-weapon training?</h3></article>
    </section>
  </main>


  <footer>&copy; 2016 Camper Cat</div>


</body>
```
#### Screenshot

![footer](https://user-images.githubusercontent.com/3828967/34631259-358d3bb2-f278-11e7-8a24-e4b0968b1c44.jpg)

**Comment by BackinGehenna** [comment]
<footer>&copy; 2016 Camper Cat</footer>
